pkg/provision/storage/commonStorage.go

```diff
@@ -144,6 +145,15 @@ func (p *CommonStorageProvisioner) rewriteContainerVolumeMounts(workspaceId, pvc
 		additionalVolumes[additionalVolume.Name] = true
 	}
 
+	// Containers in podAdditions may reference volumes defined in pod overrides, and this is not an error
+	overridesVolumes, err := overrides.GetPodVolumeOverrides(workspace)
```

Comment:
Given the usage, it might be simpler to instead have the method just return a set of volume names. 

```go
func GetVolumesFromOverrides(workspace *dw.DevWorkspaceTemplateSpec) (map[string]bool, error) {
```

I was slightly confused reading the method in `pods.go` as it makes it seem like something might require just the volumes from overrides.